### DISCUSSION TITLE
課題13についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#49] [Q&A]
14行目のjとは何でしょうか？
隣り合った数字を比べるため、jを作成し、iの値とjの値を比べるためのものでしょうか？

**Comment by nsc-kiwamu** [comment]
作成した配列をソートする中で、iの値やjの値がどのように変化していくかなど
コンソールログなどで確認できているでしょうか？

頭の中だけで理解しようとしている段階であれば、一度コンソールログに出して
動きを見てみて下さい。
その上で不明であれば、コンソールログで出力している内容をpushして
再度聞いて下さい。
